requirements.txt

```diff
@@ -0,0 +1,2 @@
+# for tests
+vcrpy==2.0.1
```

Comment:
What's the reason you picked an old version? There is now https://pypi.org/project/vcrpy/4.1.1/. If you have to specify a version at all, you could use vcrpy<5.0.0 to protect against breaking API changes in the next major release.

Reply:
I just copied it from another repo. I thought there was a reason for specifying that particular version.
Changed to `vcrpy<5.0.0`. Thanks!